app/services/finance/npq/statement_calculator.rb

```diff
@@ -22,6 +22,12 @@ def total_output_payment
         end
       end
 
+      def total_targeted_delivery_funding
```

Comment:
can we add a unit test for this as well?

Reply:
Added test for method